Add kube-state-metrics as third party image (#268)
kyma-project/third-party-images: kube-state-metrics/README.md

# KSM Docker Image

The official KSM Docker image is [here](https://github.com/kubernetes/kube-state-metrics/blob/master/Dockerfile).

> This custom image has been created to mitigate the security vulnerabilities the official image has.

Reviewer: ```suggestion
This custom image mitigates the security vulnerabilities that the official image has.
```
